libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserUtils.java

```diff
@@ -54,9 +94,15 @@ public static Map<String, Policy> createPluginPolicies(Collection<PluginData> pl
             String pluginName = pluginRoot.getFileName().toString();
             final Set<String> moduleNames = getModuleNames(pluginRoot, entry.isModular());
 
-            var overriddenPolicy = parsePolicyOverrideIfExists(overrides, version, entry.isExternalPlugin(), pluginName, moduleNames);
-            if (overriddenPolicy.isPresent()) {
-                pluginPolicies.put(pluginName, overriddenPolicy.get());
+            var overriddenPolicy = parsePolicyOverrideIfExists(
+                overrides.get(pluginName),
+                version,
+                entry.isExternalPlugin(),
+                pluginName,
+                moduleNames
+            );
+            if (overriddenPolicy != null) {
```

Comment:
Nit: again with the double-negatives.

Reply:
I will change this in a follow-up, this is going to be merged too.